src/views/Header.js

```diff
@@ -123,6 +123,7 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
         pressColorAndroid={options.headerPressColorAndroid}
         tintColor={options.headerTintColor}
         title={backButtonTitle}
+        truncatedTitle={options.headerTruncatedBackTitle}
```

Comment:
Actually I believe we should read it from precious scene, just like we do for back title. Didnt notice that while reviewing. Mind creating a new PR to update it?